src/TodoApp.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import { Header } from './components/Header';
+import { Main } from './Main/Main';
+import { Filter } from './components/Filter';
+import { useLocalStorage } from './utils/hooks/useLocalStorage';
+import { RegistrationForm } from './components/RegistrationForm';
+
+export const TodoApp: React.FC = () => {
+  const [todos, setTodos] = useLocalStorage('todos', []);
+  const [isProcessing, setIsProcessing] = useState<number[]>([]);
```

Comment:
The name of state is unclear here, pls change it
names starting with _iS_ should only be used for boolean values